src/Orleans/Orleans.csproj

```diff
@@ -74,8 +74,9 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Microsoft.CSharp" />
-    <Reference Include="Newtonsoft.Json">
-      <HintPath>$(SolutionDir)packages\Newtonsoft.Json.6.0.8\lib\net45\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```

Comment:
@jthelin If this is to be our official policy on how we handle nugget references, can you provide some sort of justification for this; in docs, or as an issue?  Something a bit more convincing then "it is evil" :).  What binding problems does this lead to?  The thing here, is that we're asking all contributors to do this extra work, so we should have some meaningful explanation of why we expect this of them.  Additionally, we'll likely be answering this question over and over, so having a link with a detailed explanation would be quite handy.